external-crates/move/crates/move-analyzer/src/bin/move-analyzer.rs

```diff
@@ -52,6 +52,9 @@ fn main() {
 
     let (connection, io_threads) = Connection::stdio();
     let symbols = Arc::new(Mutex::new(symbols::empty_symbols()));
+    let pkg_deps = Arc::new(Mutex::new(
```

Comment:
This has been moved here from `symbols.rs` because we now need it both for in the asynchronous symbolicator and to speed up symbol computation before completion request is fulfilled (and these are done in different threads).